README.md

```diff
@@ -1160,6 +1160,7 @@ See also _[Monitoring](#monitoring)_
 - [Homer](https://github.com/bastienwirtz/homer) - A dead simple static homepage to expose your server services, with an easy yaml configuration and connectivity check. `Apache-2.0` `HTML5`
 - [iDashboard-PHP](https://github.com/causefx/iDashboard-PHP) - HTPC Dashboard to load website services. `MIT` `PHP`
 - [Organizr](https://github.com/causefx/Organizr) - Organizr aims to be your one stop shop for your Servers Frontend. `GPL-3.0` `PHP`
+- [Personal management system](https://github.com/Volmarg/personal-management-system) - Central point for managing personal data (billings, payments, job holidays, notes etc.) ([Demo](http://185.204.216.136:8000/)), `MIT` `PHP`
```

Comment:
- Missing final dot after description
- Extra comma after demo link
- Can you host the demo at a proper domain name?

Reply:
FYI the demo does not work right now for me.

Also https would be awesome 😉

Reply:
Yes the demo died becauase i just added domain to it and i have acces only to terminal on smartphone now so had issues to configure it but works now under New url - will add it to pull in moment.